### ISSUE TITLE
Foundry v10 Testing (Build 279). 10000s compatibility errors generated when manipulating JEs

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#100]
```
commons.js:1243 Error: You are accessing the NoteDocument#data object which is no longer used. 
Since V10 the Document class and its contained DataModel are merged into a combined data structure. 
You should now reference keys which were previously contained within the data object directly.
    at logCompatibilityWarning (commons.js:1232:19)
    at NoteDocument._logV10CompatibilityWarning (commons.js:6119:14)
    at get data [as data] (commons.js:6019:24)
    at QuickEncounter.js:1085:74
    at Array.find (<anonymous>)
    at QuickEncounter.getEncounterScene (QuickEncounter.js:1085:60)
    at QuickEncounter.displayQEDialog (QuickEncounter.js:1609:40)
    at Object.onRenderJournalPageSheet [as fn] (QuickEncounter.js:1587:28)
    at #call (foundry.js:688:20)
    at Hooks.callAll (foundry.js:645:17)
```

**Comment by spetzel2020** [comment]
This may also be resulting in looping and bugs (like cannot create a new Journal Page with embedded Actors)

**Comment by spetzel2020** [comment]
Fixed in 1.0.4i
(Seemed to be related to #92 and #93 )

**Comment by spetzel2020** [comment]
Released in 1.0.4
